container-core/src/main/java/com/yahoo/container/jdisc/HandlerEntryPoint.java

```diff
@@ -0,0 +1,19 @@
+package com.yahoo.container.jdisc;
+
+import java.net.URI;
+
+public class HandlerEntryPoint {
```

Comment:
Consider adding optional description and/or category to be listed with the handler URIs. This could also be used to aggregate the links somehow.

Reply:
We want more information here as we go, but for now I just want to get out the URLs.